Update decision-tree.md adding links (#36)
w3c/wai-arrm: content/content-author.md

{% include box.html type="start" h="2" title="Summary" class="full" %}
{:/}

> This page lists some digital accessibility tasks that are usually the responsibility of [@@role].

Reviewer: ```suggestion
This page lists some digital accessibility tasks that are usually the responsibility of content authors.
```

**Example job titles for this role**

Content Strategist, Content Creator, Content Designer, Content Author, Digital Copywriter, UX Writer, Content Producer, Technical Writer, Information Developer, Content Developer.

## Tasks to get started

> Below is a list of tasks for content authors to get started making your work more accessible to disabled people. If these design tasks aren't met, your content can create barriers to users with disabilities.

Reviewer: ```suggestion
Below is a list of tasks for content authors to get started making your work more accessible to disabled people. If these tasks aren't met, your content can create barriers to users with disabilities.
```

[Read Lakshmi's full story](https://www.w3.org/WAI/people-use-web/user-stories/story-three/) and learn about other design tasks that benefit users like her.

> ### Additional Resources

Reviewer: ```suggestion
### Additional resources
```
